config/config.json

```diff
@@ -1,21 +1,21 @@
 {
   "development": {
     "username": "root",
-    "password": null,
+    "password": "mysqlpassword",
```

Comment:
Is there a reason for this? It is better to leave it to `null`, since most developers do not put a password on their development database.

Reply:
no particular reason for this, I'll change it to null